src/main/resources/application.properties

```diff
@@ -14,4 +14,21 @@ api.v1.base_path=/api/v1
 
 springdoc.pathsToMatch=${api.v1.base_path}/**
 
-logging.pattern.console=${LOG_PATTERN:%d{dd-MM-yyyy HH:mm:ss.SSS} %magenta([%thread]) %highlight(%-5level) %logger.%M - %msg%n}
+spring.cache.jcache.provider=com.github.benmanes.caffeine.jcache.spi.CaffeineCachingProvider
+spring.cache.cache-names=buckets
+spring.cache.caffeine.spec=maximumSize=10000,expireAfterAccess=60s
+
+bucket4j.enabled=${ENABLE_RATE_LIMITING:false}
+bucket4j.filters[0].cache-name=buckets
+bucket4j.filters[0].filter-method=servlet
+bucket4j.filters[0].filter-order=1
+bucket4j.filters[0].url=/api/v1/.*
+bucket4j.filters[0].http-response-body={ "status" : 429, "error": "Too many requests", "message" : "You have exhausted your API Request Quota" }
+bucket4j.filters[0].rate-limits[0].bandwidths[0].capacity=1
+bucket4j.filters[0].rate-limits[0].bandwidths[0].time=1
```

Comment:
will it only be 1 request per minute?

Reply:
No it will be 20 requests per minute currently. I was doing some testing and set it to 1 and forgot to revert. Changed it back now.